awscli/customizations/cloudformation/deploy.py

```diff
@@ -333,7 +333,7 @@ def deploy(self, deployer, stack_name, template_str,
         try:
             if v2_debug and fail_on_empty_changeset:
                 uni_print(
-                    'AWS CLI v2 UPGRADE WARNING: In AWS CLI v2, '
+                    '\nAWS CLI v2 UPGRADE WARNING: In AWS CLI v2, '
```

Comment:
Might not matter at this point, but could have been nice to wrap all of these in a `print_migration_debug_message`